sources/packages/web/src/views/institution/student/InstitutionStudentFileUploads.vue

```diff
@@ -1,3 +1,36 @@
 <template>
-  <tab-container><div>File Upload component to be built.</div></tab-container>
+  <tab-container>
+    <student-file-uploads
+      :studentId="studentId"
+      :canUploadFiles="false"
```

Comment:
these properties can be defaulted to false at the component level and not required to mention explicitly.